### ISSUE TITLE
License

### ISSUE THREAD

**Issue by probonopd** [Ultimaker/Ultimaker2Go#1]
Please clarify the license of this repository. What is one allowed to do with it?

**Comment by Ghostkeeper** [comment]
All files in this repository are licensed under CC-by-NC. It mentions this in the `LICENSE` file: https://github.com/Ultimaker/Ultimaker2Go/blob/master/LICENSE
The full license text is not mentioned in the file, but it provides a link to where it can be read. This is not ideal, but sufficient as far as I'm concerned -- and for the law too.

You can review the full terms of the license here: https://creativecommons.org/licenses/by-nc/3.0/legalcode

In summary, you can use these files for non-commercial purposes as long as you provide attribution.

**Comment by probonopd** [comment]
Thanks, that clarifies it. (I don't know why I missed the LICENSE file, it's obvious)
